src/app.jsx

```diff
@@ -65,19 +67,46 @@ const theme = createTheme({
   },
 });
 
-function App() {
+const App = () => {
+  // TO DO: uncomment and set isLoggedIn to false for production setting
+  // const [isLoggedIn, setLoggedIn] = useState(false);
+
+  // For development mode, isLoggedIn is set to true
+  const [isLoggedIn, setLoggedIn] = useState(true);
```

Comment:
we can store sessions in the database to persist login information

Reply:
will look into it